swatch-model-events/schemas/event.yaml

```diff
@@ -171,31 +214,43 @@ properties:
       - oracle
     required: false
   billing_account_id:
-    description: The account ID used for billing.
+    description: >
+      The marketplace account ID used for billing. Required for metered offerings. This is the AzureSubscriptionId for Azure and the CustomerAwsAccountId for AWS.
+      If azure_subscription_id is set then that value is used to for billing_account_id.
     type: string
     existingJavaType: java.util.Optional<String>
     required: false
+    examples:
+      - f49a1a52-7c09-4c2f-a227-c8680c7b5a17
+      - 51231513251
   azure_tenant_id:
-    description: The azure tenant ID used for billing
+    description: Deprecated. The azure tenant ID used for billing. No longer used by swatch.
     type: string
     existingJavaType: java.util.Optional<String>
     required: false
+    deprecated: true
     deprecationMessage: This field is no longer used in billing logic
   azure_resource_id:
-    description: The azure resource ID used for billing
+    description: >
+      The azure_resource_id is generally not available on service instances. azure_subscription_id is used to determine which azure resource to bill against.
     type: string
     existingJavaType: java.util.Optional<String>
     required: false
   azure_subscription_id:
-    description: The azure subscription ID used for billing
+    description: The azure subscription ID used for billing. Required for PAYG azure instances.
     type: string
     existingJavaType: java.util.Optional<String>
     required: false
+    examples:
+      - f49a1a52-7c09-4c2f-a227-c8680c7b5a17
   product_tag:
-    description: Product tag
+    description: Product tag of the associated offering.
```

Comment:
Could use a little more context around this one. What should this match in the config?